### PR TITLE
Remove CSS transition from selected line

### PR DESCRIPTION
**Pull request by mrnugget**
I want to remove this because it feels like Sourcegraph can't keep up
with the clicks. It feels slow.

In fact: when I said to someone else that I bet it's a CSS transition
why this is slow they said "nah, looks like something's loading in the
background"

### CODE REVIEW

**Comment by mrnugget** [comment]
Related ticket: https://github.com/sourcegraph/sourcegraph/issues/28901

File: client/web/src/repo/blob/Blob.module.scss [hunk 1]
```diff
@@ -38,8 +38,6 @@
     }
 
     tr {
-        transition: background 200ms ease-out;
```
Comment (olafurpg) [inline client/web/src/repo/blob/Blob.module.scss:41]: I just tried this PR locally and there still seems to be a delay when clicking on lines 🤔 Am I misunderstanding what this PR does?

CleanShot 2021-12-17 at 14 13 40

**Review by rrhyne — APPROVED** [review]
Nice, thanks for taking this on.

File: client/web/src/repo/blob/Blob.module.scss (hunk 1 above)
Comment (mrnugget) [reply client/web/src/repo/blob/Blob.module.scss:41]: Hmmm. It was gone for me locally. Let me try again.

Comment (mrnugget) [reply client/web/src/repo/blob/Blob.module.scss:41]: I think what's going on here is something else: extensions checking `git blame`? Because that is much slower than 200ms
